packages/pn-personafisica-webapp/src/utility/contacts.utility.ts

```diff
@@ -90,7 +101,7 @@ const isDropdownItemDisabled = (
 ): boolean => {
   // the address is disabled if there is an address with the same sender, already added
   const senderHasAlreadyAddress =
-    addresses[`special${allowedAddress.channelType}Addresses`].findIndex(
+    addresses[`special${allowedAddress.channelType}Addresses`]?.findIndex(
```

Comment:
Why the ?, this is should be useless

Reply:
removing the ? will cause the following error executing the contact.utility.test.ts
![image](https://github.com/user-attachments/assets/3af10fc5-9bc5-43db-bde3-0f45adbd97dc)

Reply:
The error is dued to the IO we added inside the addressesRelationships array. The tests that fail cannot find the specialAPPIOAddresses array (even if empty) and causes the app to crash. And in fact everything is fine for PG, old tests pass removing the ? (I had added it to keep some sort of uniformity between PF and PG apps).